packages/react-test-renderer/src/__tests__/ReactShallowRendererHooks-test.js

```diff
@@ -252,6 +252,36 @@ describe('ReactShallowRenderer with hooks', () => {
     expect(effectsCalled).toEqual([]);
   });
 
+  describe('when callEffects option is used', () => {
+    it('should trigger effects after render', () => {
+      let happenings = [];
```

Comment:
I called it `happenings` rather than `events` to make it obvious these are not DOM events or anything like that.